resources/release/criteria-not-met-template.md

```diff
@@ -0,0 +1,8 @@
+Hi ${RELEASE_OWNER}, </br>
+
+The below ${TYPE_OF_CRITERIA} criteria for your component has not been met. </br>
```

Comment:
How about?
`The component ${COMPONENT_NAME} is not meeting following release window criteria`?